xlive/Blam/Engine/objects/widgets/widgets.cpp

```diff
@@ -1,7 +1,20 @@
 #include "stdafx.h"
 #include "widgets.h"
 
+#include "main/interpolator.h"
+#include "objects/objects.h"
+
+
+
 void __cdecl widgets_new(datum object_index)
 {
 	INVOKE(0x14FFE2, 0x14CD7A, widgets_new, object_index);
+	object_initialize_for_interpolation(object_index);
```

Comment:
why not just move this call to a later point in object_new? cause that's where widgets_new is getting called, and you're now initializing the interpolation data twice

Reply:
Forgot to remove this, the call is useless